server/container/guice/cassandra-ldap-guice/src/test/java/org/apache/james/CassandraLdapJamesServerTest.java

```diff
@@ -80,4 +83,12 @@ void mailsShouldBeWellReceivedBeforeFirstUserConnectionWithLdap(GuiceJamesServer
             .select("INBOX")
             .awaitMessage(calmlyAwait);
     }
+
+    @Test
+    void receivingMailShouldIssueAnSmtpErrorWhenLdapIsNotAvailable(GuiceJamesServer server) {
+        ldap.getLdapRule().stop();
```

Comment:
if this test gets played before the other ones defined by the contract, does ldap container stays in the stop state?

Reply:
fixed